flowey/flowey_lib_hvlite/Cargo.toml

```diff
@@ -3,6 +3,7 @@
 [package]
 name = "flowey_lib_hvlite"
 edition = "2021"
+rust-version = "1.82"
```

Comment:
Looks like this key can be inherited via `.workspace`?
https://doc.rust-lang.org/cargo/reference/workspaces.html#the-package-table

Reply:
Oh thats interesting. Maybe we should update xtask fmt to include rust-version.workspace = true on everything? That can be a follow up though, don't want to do it now.

Reply:
Sounds like you're already working on this follow up. Approved this PR, looking forward to the next one :)